ember-cli-flash/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "ember-cli-flash",
+  "version": "4.0.0",
+  "description": "Simple, highly configurable flash messages for ember-cli",
+  "keywords": [
+    "ember-addon",
+    "ember",
+    "ember-cli",
+    "flash messages",
+    "flash",
+    "growl",
+    "toastr",
+    "notifications",
+    "notify"
+  ],
+  "homepage": "https://github.com/adopted-ember-addons/ember-cli-flash",
+  "bugs": "https://github.com/adopted-ember-addons/ember-cli-flash/issues",
+  "repository": "https://github.com/adopted-ember-addons/ember-cli-flash",
+  "license": "MIT",
+  "author": "Lauren Tan",
+  "contributors": [
+    {
+      "name": "Mike North"
+    },
+    {
+      "name": "Scott Batson"
+    },
+    {
+      "name": "Brian Runnells"
+    }
+  ],
+  "exports": {
+    ".": "./dist/index.js",
```

Comment:
This file should probably have a types entry as well

Reply:
And maybe reexport to component and service type if that's not happening already

Reply:
An index file doesn't actually exist in `/dist` (yet) so I've removed. We can add a top level module export with types as an enhancement later